composer.json

```
@@ -4,7 +4,7 @@
     "keywords": ["laravel", "backpack", "pan"],
     "license": "MIT",
     "require": {
-        "backpack/crud": "^6.0",
+        "backpack/crud": "^7.0.0-alpha",
```

Easier to support both, right?

```suggestion
        "backpack/crud": "^6.0|^7.0",
```